molecule/default/playbook.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Converge
+  hosts: all
+
+  pre_tasks:
+    - name: Install apt-transport-https, net-tools
+      become: yes
+      apt:
+        name: "{{ item }}"
+        update_cache: yes
+        cache_valid_time: 3600
+      with_items:
+        - apt-transport-https
+        - net-tools
+
+  roles:
+    - role: sansible.postgresql
```

Comment:
Maybe specify the root user and password here as well in case the default changes in the postgresql role.

Reply:
Good point!